db/db_impl/db_impl.h

```diff
@@ -2322,7 +2322,7 @@ class DBImpl : public DB {
   // If callback is non-null, the callback is refreshed with the snapshot
   // sequence number
   //
-  // `unref_only` being set to true indicates that the SuperVersions were
+  // `sv_from_thread_local` being set to false indicates that the SuperVersions
   // obtained from the ColumnFamilyData, whereas false indicates they are thread
```

Comment:
I believe the second clause needs a small update here too ("whereas true indicates...")

Reply:
Thank you for the thorough review!